src/app/settings/panel-settings/models/panel-layout-configuration.ts

```diff
@@ -0,0 +1,9 @@
+import { PanelModel, PanelType } from './panel-enums'
+
+export interface PanelLayoutConfiguration {
+  id: string
+  name: string
+  type: PanelType
+  model: PanelModel
+  // TODO: Missing inputConfigurations
```

Comment:
```suggestion
  // TODO: Missing inputConfigurations
```

Did you forget to remove this? 😄 or is it coming in another PR/SOF-issue?

Reply:
It's intentional. It's even mentioned in the PR description:
![image](https://github.com/user-attachments/assets/15f0c88c-d067-4642-93e5-f0df44b4cbae)